### DISCUSSION TITLE
Guide to Soroban CLI

### DISCUSSION THREAD

**Discussion by wildework** [stellar/sorobanathon#72] [General]
I had trouble finding an extensive list of the Soroban CLI subcommands and what they along, along with sample inputs and their outputs. This is my first attempt at cataloguing all the subcommands.

Also, I've explored the `invoke` subcommand and how it traces all the way into the `wasmi` interpreter.

[sorobandev.com/guides/soroban-cli](https://sorobandev.com/guides/soroban-cli).

Really enjoying this exploration so far :)

**Comment by achallagundla** [comment]
Heya @wildework  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1000 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
